docs/python-reference/index.md

```diff
@@ -0,0 +1,53 @@
+These are autogenerated Python references for the PrairieLearn library. These helpers are available in the Python environment when running a PrairieLearn question.
+
+You can use
+
+```python
+from prairielearn.<submodule> import ...
+```
+
+to use any of these helpers. Any code from `prairielearn.internal` should be considered internal and may change without notice.
+
+!!! warning
+
+    The following imports are now deprecated.
+
+    === "Old"
+
+        ```python
+        import colors
+        import to_precision
+        import python_helper_sympy
+        ```
+
+    === "New"
+
+        ```python
+        import prairielearn.colors
+        import prairielearn.to_precision
+        import prairielearn.sympy_utils
+        ```
+
+## Element Development
+
+These are the core classes for developing elements in PrairieLearn.
+
+<!-- prettier-ignore -->
+::: prairielearn.core
+    options:
+        members:
+            - from_json
+            - to_json
+            - QuestionData
+            - PartialScore
+
+### SymPy
+
+These utilities are for working with SymPy.
+
+<!-- prettier-ignore -->
+::: prairielearn.sympy_utils
+    options:
+        members:
+            - convert_string_to_sympy
```

Comment:
I'm not a fan of the need to horizontally scroll here:
![image](https://github.com/user-attachments/assets/d36c8813-ba5d-4727-ab42-c1f163ffee58)

I would rather have the signature formatted the way that ruff formats it in the file instead of on a single line like this.

Reply:
Counterpoint: we have a whole table describing these right below it.

Reply:
This is fixed in the latest commit.

Reply:
![CleanShot 2025-02-07 at 23 50 21@2x](https://github.com/user-attachments/assets/59ae5a58-afe0-43f4-9ca0-7ce7c80a68bb)
I can't seem to replicate this locally by doing a build + serve.

Reply:
Not sure whats different about the two environments though?